File: 育成中コンテンツ/広告/アドフリくん/adfurikunSDK_Android_moviereward_3.20.0/SampleJava/app/src/main/java/sample/adfurikunsdk/jp/samplejava/SampleApplication.java
package sample.adfurikunsdk.jp.samplejava;

import android.app.Application;
import jp.tjkapp.adfurikunsdk.moviereward.AdfurikunSdk;

public class SampleApplication extends Application {

    @Override
    public void onCreate() {
        super.onCreate();
        AdfurikunSdk.init(this);
    }
}

File: 広告/アドフリくん/adfurikunSDK_Android_moviereward_3.20.0/SampleKotlin/app/src/main/java/jp/glossom/sample/samplekotlin/SampleApplication.kt
package jp.glossom.sample.samplekotlin

import android.app.Application
import jp.tjkapp.adfurikunsdk.moviereward.AdfurikunSdk

open class SampleApplication : Application() {

    override fun onCreate() {
        super.onCreate()

        AdfurikunSdk.init(this)
    }
}